pages/index.js

```diff
@@ -412,12 +412,17 @@ export default function Home({ total, today }) {
                     <div className="flow-root rounded-lg bg-gray-50 px-6 pb-8">
                       <div className="-mt-6">
                         <div>
+                        <Link 
+                          href={feature.path} 
+                             >
                           <span className="inline-flex items-center justify-center rounded-xl bg-indigo-500 p-3 shadow-lg">
+                            
                             <feature.icon
                               className="h-8 w-8 text-white"
                               aria-hidden="true"
-                            />
+                                 />    
```

Comment:
I'd say leave the formating for this line, it seems indented one too many.

Reply:
@HarshDeep61034 hey, are you editing directly on GitHub?

Reply:
No

Reply:
> No

Oh okay, then maybe you forgot to format ig, or don't have prettier installed?

Reply:
yeah i forgot to format.